### DISCUSSION TITLE
Dextensify | Disable admin-enforced chrome extensions with no bookmarklets or crosh LTBEEF v2

### DISCUSSION THREAD

**Discussion by blueiicey** [3kh0/ext-remover#704] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
Hello! I have used your website for lots of extension disablers.
But since ltbeef was patched  on chrome v115 and my district used blocksi to disable bookmarklets, all of the exploits listed were patched for me.
I would highly suggest adding https://dextensify.pages.dev/ to the website. It is a variation of LTBEEF that isn't patched. It works on GoGuardian, Blocksi, Gopherbuddy, Securly and more.
Thanks in advance!

**Comment by darkdragon1076** [comment]
dexrensify dosen't work. the two button aren't there

**Reply by blueiicey** [reply]
Dextensify is working, can you give me details?

**Reply by sayreobryan** [reply]
Make sure that you're using chrome.

**Reply by LordofDarkenedShadow** [reply]
my dextify html when i download it is blocked for some reason

**Reply by TheAnimemer** [reply]
Talk about reviving an ancient thread, wow. I’ve graduated highschool now LOL

**Comment by TheAnimemer** [comment]
My chromebook crashes when i try to disable Lightspeed, any idea how to fix that?
It also seems like this method is only temporary? Do you have to keep going back to the extension settings every few minutes until you're done with it or does it eventually work on its own?

**Reply by c0nstantly** [reply]
Also my extension does not have an "allow access to file URLs" switch so the exploit doesn't work.
